runtime/onert/api/src/nnfw_api_internal.cc

```diff
@@ -205,6 +205,36 @@ NNFW_STATUS nnfw_session::run()
   return NNFW_STATUS_NO_ERROR;
 }
 
+NNFW_STATUS nnfw_session::run_async()
+{
+  if (!isStatePreparedOrFinishedRun())
+  {
+    std::cerr << "Error during nnfw_session::run_async : "
+              << "run_async should be run after prepare" << std::endl;
```

Comment:
message like "run_async should be run after prepare or finished running async" be more specific. :-)